packages/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -860,7 +860,7 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
                   )}
                   onMouseOut={this._handleMouseOut}
                   strokeWidth={0}
-                  role="none"
+                  role="img"
```

Comment:
Setting this role for the dummy magnetic callout circle caused the accessibility test to fail.
Referred the below image for the change:
<img width="498" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/92f2a93f-b96f-4274-9bb5-c498eb2a95c6">

Reply:
is this an issue that you are fixing or introduced to test the accessibility tests.

Reply:
This dates back to the magnetic callout PR for LineCharts. We had to set the role for the dummy circle as none. But this violates accessibility rules and fails to pass the test

Reply:
Should we separate this fix from this PR?

Reply:
@krkshitij could you suggest what aria role to assign to the dummy circles that are used just for increasing the hover surface area for line chart but don't have any other purpose.

Reply:
@Shubhabrata08 You can remove the `role` and `aria-label` attributes from the dummy circle. We don't want it to be read by screen readers because the actual circle is already accessible.

Reply:
Ok